app/controllers/users.js

```diff
@@ -39,25 +48,38 @@ const signIn = async (req, res, next) => {
       throw errors.unauthorizedError(`The user ${email} is wrong`);
     }
 
-    const isThesamePassword = await utilities.compareEncryptText(password, user.password);
+    const isTheSamePassword = await utilities.compareEncryptText(password, user.password);
 
-    if (!isThesamePassword) {
+    if (!isTheSamePassword) {
       throw errors.unauthorizedError('The password is wrong');
     }
 
-    const userSerialized = signInSerializer(user);
+    const userSerialized = userObjectSerializer(user);
     const { token, expires } = utilities.generateToken(userSerialized);
 
     logger.info(`user with email ${user.email} has logged in successfully`);
 
-    return res.status(201).send({ token, expires });
+    return res.status(200).send({ token, expires });
   } catch (error) {
     logger.error(`userController::signIn::error::${error}`);
     return next(error);
   }
 };
 
+const getUsers = async (req, res, next) => {
+  try {
+    const { offset, limit } = req.query;
+    let users = await UserService.getUsers(limit, offset);
```

Comment:
I didn't see this `UserService` in previous prs, but PascalCase is reserved for clases, it should not be used for non-clases

Reply:
Clases or models